test/roundtrip/named-type.txt

```diff
@@ -0,0 +1,8 @@
+;;; TOOL: run-roundtrip
+;;; ARGS: --stdout --debug-names
+(module $a
+  (type $b (func)))
+(;; STDOUT ;;;
+(module $a
+  (type $b (func)))
```

Comment:
Sorry, can we just update the existing `test/roundtrip/debug-names.txt` test?

Reply:
Sure, how about this?